go/vt/schema/online_ddl_test.go

```diff
@@ -274,11 +275,13 @@ func TestNewOnlineDDLs(t *testing.T) {
 		"create index i_idx on t(id, `ts`, name(12))":   {sqls: []string{"alter table t add index i_idx (id, ts, `name`(12))"}},
 		"create unique index i_idx on t(id)":            {sqls: []string{"alter table t add unique index i_idx (id)"}},
 		"create index i_idx using btree on t(id)":       {sqls: []string{"alter table t add index i_idx (id) using btree"}},
+		"create view v as select * from t":              {sqls: []string{"create view v as select * from t"}, isView: true},
+		"alter view v as select * from t":               {sqls: []string{"alter view v as select * from t"}, isView: true},
+		"drop view if exists v":                         {sqls: []string{"drop view if exists v"}, isView: true},
 		"create index with syntax error i_idx on t(id)": {parseError: true},
 		"select * from t":                               {notDDL: true},
 		"drop database t":                               {notDDL: true},
 		"truncate table t":                              {isError: true},
-		"drop view t":                                   {isError: true},
```

Comment:
This case isn't covered now anymore in the above tests, only the version with `if exists`, is that correct?

Reply:
Right. Just added it; the nuance of `IF EXISTS` is more in the `sqlparser`'s domain, where it is more thoroughly tested, but why not have an extra test here as well.